src/zope/app/file/browser/file.py

```diff
@@ -293,7 +305,7 @@ def update_object(self, data, contenttype):
         # Update *only* if a new value is specified
         if data:
             self.context.data = data
-            descriptor.attributes += "data",
+            descriptor.attributes += ("data",)
```

Comment:
It is confusing -- my LOL was prompted by me not noticing there was a trailing comma in the original, and assuming the old code had a bug.